components/RelatedItems.vue

```diff
@@ -133,10 +134,14 @@ export default Vue.extend({
     try {
       this.refreshItems();
     } catch (error) {
-      console.error('Unable to get related items:', error);
+      this.pushSnackbarMessage({
```

Comment:
I agree with this, but I'm concerned that we're losing potentially valuable debug logs.

We have a similar issue with the current web client, which swallows every error it produces, and makes it very annoying to track things.

Most of the try...catch blocks have "breaking failures", as in reach the catch block breaks functionality in the app. As such, I think we could maybe try to build a modal that gives a traceback to the user or the developer, which could then be used to help debugging.

Just a though.

Reply:
I think that for content such as:

 - related items,
 - Cast and Crew
 - Content in a home section

A message can just be displayed in the snack bar

For failures such as:

 - unable to play content
 - unable to get item details on item details page

A full screen message should show as these are *severe* failures, whereas the other ones are not.

Reply:
Agreed, that's what I had in mind :) ("blocking" things would generate a modal with the trace, non-blocking would either show a snackbar or be handled otherwise).